src/applications/vaos/components/ConfirmedAppointmentListItem.jsx

```diff
@@ -27,7 +29,11 @@ export default function ConfirmedAppointmentListItem({ appointment }) {
             {' '}
             Where{' '}
           </h3>
-          {getAppointmentLocation(appointment)}
+          {isVideoVisit(appointment) ? (
+            <VideoVisitLink appointment={appointment} />
+          ) : (
+            <>{getAppointmentLocation(appointment)}</>
```

Comment:
Do we need the `<>` wrapping here? Could it be `getAppointmentLocation(appointment)`, without the `<></>` or the `{}`?